datalad/support/sshconnector.py

```diff
@@ -233,13 +233,20 @@ def is_open(self):
     def open(self):
         """Opens the connection.
 
-        In other words: Creates the SSH controlmaster to be used by this
+        In other words: Creates the SSH ControlMaster to be used by this
         connection, if it is not there already.
 
         Returns
         -------
         bool
-          Whether SSH reports success opening the connection
+          True when SSH reports success opening the connection, False when
+          the connection when a ControlMaster for an open connection already
+          exists.
```

Comment:
Can't really understand from reading for false

Reply:
"when the connection" should be removed, I guess.

Reply:
Fixed, thx!